pom.xml

```diff
@@ -243,6 +243,12 @@
         <!-- Carbon kernel version -->
         <carbon.kernel.registry.imp.pkg.version>[1.0.1, 2.0.0)</carbon.kernel.registry.imp.pkg.version>
         <carbon.kernel.version>4.4.7</carbon.kernel.version>
+        <carbon.kernel.package.import.version.range>[4.4.0, 5.0.0)</carbon.kernel.package.import.version.range>
+
+        <!--Carbon identity version-->
+        <identity.framework.version>${project.version}</identity.framework.version>
```

Comment:
This property is not used. We can remove this. And as mentioned in the above comment, refrain from using "project.version" for the dependencies coming from an external repo.